.github/workflows/release-sync.yml

```diff
@@ -0,0 +1,157 @@
+name: Sync Release with Master
+
+# Triggered when a new version is released publicly
+on:
+  workflow_dispatch:
+  release:
+    types: [ released ]
+
+jobs:
+  create-sync-pull-request:
+    name: Create release sync PR
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ secrets.RELEASE_TOKEN }}
+      ORG_NAME: HathorNetwork
+      PROJECT_NUMBER: 15
+      COLUMN_NAME: "In Progress (WIP)"
```

Comment:
Why not In Progress (Done)?

Reply:
Good point! Changed in https://github.com/HathorNetwork/hathor-wallet-lib/pull/692/commits/92cfca0f48f294aadb2c0be7d2cedd78e8550d9c